great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -579,16 +577,17 @@ def get_domain_records(  # noqa: C901 - 17
 
         return data
 
+    @public_api
     def get_compute_domain(
         self,
         domain_kwargs: dict,
         domain_type: Union[str, MetricDomainTypes],
         accessor_keys: Optional[Iterable[str]] = None,
     ) -> Tuple[pd.DataFrame, dict, dict]:
-        """
-        Uses the given domain kwargs (which include row_condition, condition_parser, and ignore_row_if directives) to
-        obtain and/or query a batch.  Returns in the format of a Pandas DataFrame. If the domain is a single column,
-        this is added to 'accessor domain kwargs' and used for later access
+        """Uses the given domain kwargs (which include row_condition, condition_parser, and ignore_row_if directives) to obtain and/or query a batch.
+
+        Returns in the format of a Pandas DataFrame along with Domain arguments required for computing.  If the Domain
+        is a single column, this is added to 'accessor domain kwargs' and used for later access.
 
         Args:
             domain_kwargs (dict) - A dictionary consisting of the domain kwargs specifying which data to obtain
```

Comment:
Note, check the rendering here. these may need to use a colon after the arg name - e.g.:
```
Args:
    domain_kwargs: A dictionar..... etc etc.
```